claat/cmd/update.go

```diff
@@ -69,14 +70,18 @@ func CmdUpdate(opts CmdUpdateOptions) {
 			ch <- &result{d, meta, err}
 		}(d)
 	}
+
+	errs := []error{}
 	for range dirs {
 		res := <-ch
 		if res.err != nil {
-			errorf(reportErr, res.dir, res.err)
+			errs = append(errs, res.err)
+			log.Printf(reportErr, res.dir, res.err)
 		} else {
 			log.Printf(reportOk, res.meta.ID)
 		}
 	}
+	return len(errs) == 0
```

Comment:
same comment, wonder if you should return the err array or the first error found